andela/Titan-Devs
# #163518685 Mock SendGrid Mail

#### What does this PR do?
Disable sending reset password email from test
#### Description of Task to be completed?
```POST /api/users/reset_password```
#### How should this be manually tested?
-Clone the repo and check out `bg-moch-sendgrid-mail-163518685` branch
- Go under ```helpers``` directories, and change dummyUser email to your own working email.
- Run ```yarn test```, and check your email, you will see nothing. 
#### Any background context you want to provide?
before running anything, ensure packages are updated ```yarn``` 
#### What are the relevant pivotal tracker stories?
163518685
